dom/events/relatedTargets.window.js

```diff
@@ -0,0 +1,50 @@
+const host = document.createElement("div"),
```

Comment:
I was not familiar with this style of writing a test. Looks nice to avoid boilerplate in the HTML.
Can we add metadata, especially corresponding spec URL as a comment?

Reply:
You mean link to https://dom.spec.whatwg.org/#concept-event-dispatch? Feels a little self-explanatory, but I could do.

Reply:
Yeah, I think it would be nice.